qualtran/bloqs/reflection_using_prepare.py

```diff
@@ -74,9 +77,15 @@ def control_registers(self) -> Tuple[Register, ...]:
     def selection_registers(self) -> Tuple[Register, ...]:
         return self.prepare_gate.selection_registers
 
+    @cached_property
+    def extra_reg_objects(self) -> Tuple[Register, ...]:
+        return tuple([Register(name, QAny(bitsize)) for name, bitsize in self.extra_registers])
```

Comment:
Not needed after we change the class attribute to directly store `Tuple[Register, ...]`